sources/packages/backend/apps/queue-consumers/src/processors/assessment/_tests_/cancel-application-assessment.processor.e2e-spec.ts

```diff
@@ -32,6 +33,9 @@ import {
   StudentAssessmentStatus,
 } from "@sims/sims-db";
 import * as faker from "faker";
+import { AssessmentSequentialProcessingService } from "@sims/services";
+import { TestingModule } from "@nestjs/testing";
+import { QueueConsumersModule } from "../../../../src/queue-consumers.module";
 
 jest.setTimeout(1200000);
```

Comment:
Not part of this PR, but would you mind removing this `jest.setTimeout(1200000);`?

Reply:
Thanks ❤️